cmd/zed/dev/indexfile/lookup/command.go

```diff
@@ -18,8 +18,8 @@ var Lookup = &charm.Spec{
 	Usage: "lookup -k key[,key...] index",
 	Short: "lookup a key in a Zed index file and print value as zng record",
 	Long: `
-The lookup command locates the specified key(s) in the base layer of a
-Zed index file and displays the result as a zng record.
+The "zed indexfile lookup" command locates the specified key(s) in the base layer of a
```

Comment:
```suggestion
The "zed indexfile lookup" command locates the specified keys in the base layer of a
```

Reply:
Indexes usually have one key so I think its confusing to refer to keys generally but will change.